services/paymaster/src/constraint.rs

```diff
@@ -730,9 +731,32 @@ pub fn process_compute_budget_instructions(
     Ok(priority_fee)
 }
 
+/// The Solana precompile programs that verify signatures.
+pub const PRECOMPILE_SIGNATURE_PROGRAMS: &[Pubkey] = &[
+    ed25519_program::ID,
+    secp256k1_program::ID,
+    secp256r1_program::ID,
```

Comment:
Fogo does not have this precompile live on testnet, but

1. any transactions invoking this address as a program should fail
2. the current calculation would overestimate the gas spend which is the safer path (sans checking whether the precompile is active)